src/fields/abstractField.js

```diff
@@ -60,52 +60,66 @@ export default {
 	methods: {
 		validate(calledParent) {
 			this.clearValidationErrors();
+			let validateAsync = objGet(this.formOptions, "validateAsync", false);
+
```

Comment:
Why need the `validateAsync` option? Is it not enough if we check the type of result of `validator` function (Promise or not)?

Reply:
The updated code returns a promise if validateAsync is true, and waits for async validation to complete ... this is not what the current code does, so the new option is a way to enable it while maintaining backwards compatibility so we don’t break forms that expect the validate function to return a Boolean.

Reply:
V3 would likely remove this option and always return a promise from validate... making it the default validation mode, which is what most people are likely expecting from the current async validation.

Reply:
Ok. in v3 we change it to easier (handle all as Promise)